src/main/groovy/com/jaredsburrows/checkerframework/CheckerPlugin.groovy

```diff
@@ -79,25 +89,23 @@ final class CheckerPlugin implements Plugin<Project> {
     }
 
     // Apply checker to project
-    project.gradle.projectsEvaluated {
-      project.tasks.withType(AbstractCompile).all { compile ->
-        compile.options.compilerArgs = [
-          "-processorpath", "${project.configurations.checkerFramework.asPath}".toString(),
-          "-Xbootclasspath/p:${project.configurations.checkerFrameworkAnnotatedJDK.asPath}".toString()
-        ]
-        if (!userConfig.checkers.empty) {
-          compile.options.compilerArgs << "-processor" << userConfig.checkers.join(",")
-        }
+    project.tasks.withType(AbstractCompile).all { compile ->
+      compile.options.annotationProcessorPath = project.configurations.checkerFramework
+      compile.options.compilerArgs = [
+        "-Xbootclasspath/p:${project.configurations.checkerFrameworkAnnotatedJDK.asPath}".toString()
```

Comment:
`bootClasspath` should also be preferred but is not present in the older versions of gradle currently supported